pkg/disk/type_bsd.go

```diff
@@ -19,9 +19,9 @@
 package disk
 
 // getFSType returns the filesystem type of the underlying mounted filesystem
-func getFSType(fstype [16]int8) string {
-	b := make([]byte, len(fstype[:]))
-	for i, v := range fstype[:] {
+func getFSType(fstype []int8) string {
```

Comment:
previously `fstype` is exact size.  is it changed now?

Reply:
Yes we don't it to be. Solaris has [32]int8 so made this function more generic.

Reply:
OK. Now this function is doing generic `int8SliceToString()`.  This change can be a separate PR.